Cargo.toml

```diff
@@ -69,6 +68,8 @@ rand_xoshiro = "0.6.0"
 ringlog = "0.1.0"
 serde = "1.0.152"
 serde_json = "1.0.91"
+signal-hook = "0.3.15"
+signal-hook-tokio = {version = "0.3.1", features = ["futures-v0_3"]}
```

Comment:
Seems like this isn't being used? Can we remove it?

Reply:
oops yeah I had forgotten to push up the commit that got rid of it